model/tasks/AbstractBookletTask.php

```diff
@@ -116,6 +116,7 @@ protected function generatePdf()
 
             $config = $this->getBookletConfig($instance);
             $config[BookletConfigService::CONFIG_URI] = $uri;
+            $config[BookletConfigService::CONFIG_TITLE] = $this->getParam('label');
```

Comment:
should this be escaped or similar , can it create a injection of any sort
also should we relay on the input ?

Reply:
IMO this is an internal task, not a user facing class (like a controller, request handler, etc). So we are assuming the parameters here are reliable, but... I do not know if we validate the label when we receive the user request. If we do so, we can rely on this value. If not IMO we should sanitize things on user facing layers, not here. Otherwise we would have to sanitize all in every place.

@shpran do you know those tasks parameters are validated previously? 

Other option is to sanitize on the template generation part only. 

Wdyt?

Reply:
because this is generating a output that can be injected i think it is safer to filter out as a string then not to do anything , 
the plugin is generating a pdf from html and as we know anything can be hijacked  so i would like to be 100% sure 
thank why the sanitation request

Reply:
I agree to sanitize, my point was the place we are doing. This param came from the user in other layer, so it should be validated/sanitized there, not here IMO. Or it should be sanitized during the PDF generation process. I.e `PdfBookletExporter` class.